### ISSUE TITLE
Does the API work and / or how to use it ?

### ISSUE THREAD

**Issue by Playsky** [cobinhood/api-public#2]
Hello !

I really want to use the Cobinhood's api but I have a lot of problems and questions..

I use python, and I simply want to make a request to have for example "All Currencies"
So the url should be: https://api.cobinhood.com/v1/market/currencies

So here is my Python code:
```
import requests

all_currencies = r.get("https://api.cobinhood.com/v1/market/currencies")
print(all_currencies.text)
```

So first of all, when I run this code I have the "ssl bad handshake" error ( very curious... )
So to avoid this error, I added the parameter: `verify = False` to my request
But now, the HTTP response code is 500, and I have this JSON: 

> {"success":false,"error":{"error_code":"resource_not_found"},"request_id":"4cbb6a1bf395ca23"}

And I obtain this error no matter the url..

So please can you just give me an exemple of request ? Maybe my header is not good or another thing I really don't understand how this api works, or if it's working today..

I also have another question, how can I be authenticated with the API ? Because it is said that there is an API console page in Cobinhood but I never found it..

Thanks, and I'm very sorry for my bad english !

**Comment by tommygaessler** [comment]
+1 I cannot find the "API console page" either. 

@Playsky I am guessing your request is not working since you have not authenticated.

**Comment by DSoane** [comment]
+1 I am also having trouble logging in to the API. Also get SSL error 525 - Bad Handshake.

I am writing my bot in C++ and using openssl for the first time, and it could be I am doing something wrong there. Am I right in saying that I must use the CobinHood certificate, downloaded from the site, to verify the connection? Do I need any sort of certificate of my own?

**Comment by Playsky** [comment]
@tommygaessler I'm not sure because if we have to be authenticated, there is the 

> [Authentication Required]

next to the url of the request, and there isn't it to get all currencies..

**Comment by richardcwyo** [comment]
hi guys, sorry that the doc and website are outdated right now. Team has been working some updates, fixes, and improvement right now and ETA next week to get the api doc done.

Richard

**Comment by DSoane** [comment]
@tommygaessler - Is this the page you're looking for?

https://cobinhood.github.io/api-public/

**Comment by i5starjordan** [comment]
'COBINHOOD uses JWT for APIs that require authentication. JWT header field name is authorization. The JWT can be generated and revoked on COBINHOOD exchange API console page.'

Anyone know when this API console page will be available? I can't find it.

**Comment by allenfrostline** [comment]
Just found some issues about the API page. But I believe waiting for another week won't hurt 👍 Hope everything about the API will work fine then.

**Comment by Playsky** [comment]
API requests work now ! 👍 
But, a date for the authentication and the API console page ??

**Comment by DSoane** [comment]
Yes, I have now got the live REST API to work as well. I can get the list of currencies, anyway.

However, when I try the sandbox REST API, I get an SSL 525 error. Which is what I used to get with the live API.

And as for Websockets, with live I get error 501 - unable to process request, and with the sandbox I also get SSL error 525.

Has anyone got these other three options to work?

**Comment by tommygaessler** [comment]
@richardcwyo when will we have access to the api console page and authentication? Its been more then 2 weeks.

**Comment by ecamaw** [comment]
Any news about the API console page an JSON web token generation?
Support headed me directly to the API docs but there is no further information. Many thanks!

**Comment by hbock-42** [comment]
Hello guys, any knews about the API console page?

Tanks a lot

**Comment by tommygaessler** [comment]
@hbock-42 It is live! Sign in and click API Key in the menu! https://cobinhood.com/api

**Comment by OtenMoten** [comment]
Hey guys,

I had read your discussion. 
Is everything fine now? I'm heading for writing a "Volumeter" for the Cobinhood exchange.

Is there any special behaviour of the API I should know?

Kind regards
